docs/resources/jwt_issuer.md

```diff
@@ -0,0 +1,58 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "cockroach_jwt_issuer Resource - terraform-provider-cockroach"
+subcategory: ""
```

Comment:
Do we need a subcategory? I've never seen this type of generated reference in a PR before, so apologies if it's a silly question.

I know this is a generated file, so these suggestions apply to the source. It's just simpler for me to give them here.

Reply:
Looks like we don't have subcategories as of today - https://registry.terraform.io/providers/cockroachdb/cockroach/latest/docs.
Similarly, we don't have subcategories for the API Docs - https://www.cockroachlabs.com/docs/api/cloud/v1#overview.

We had discussed subcategories in the context of APIs, and had concluded to revisit it later, once we have logical grouping to categorize the endpoints (or resources in the case of Terraform).